Sources/Sentry/SentryReachability.m

```diff
@@ -135,7 +135,7 @@
 {
     SENTRY_LOG_DEBUG(
         @"SentryConnectivityCallback called with target: %@; flags: %u", target, flags);
-#    if defined(TEST) || defined(TESTCI) || defined(DEBUG)
+#    if defined(TEST) || defined(TESTCI) | wai | defined(DEBUG)
```

Comment:
`m`: `wai`? A leftover that should be ditched?

Reply:
```suggestion
#    if defined(TEST) || defined(TESTCI) || defined(DEBUG)
```